.murdock

```diff
@@ -116,12 +132,12 @@ compile() {
     print_worker
 
     # sanity checks
-    [ $# -ne 2 ] && error "$0: compile: invalid parameters (expected \$appdir \$board)"
+    [ $# -ne 2 ] && error "$0: compile: invalid parameters (expected \$appdir \$board:\$toolchain)"
     [ ! -d "$appdir" ] && error "$0: compile: error: application directory \"$appdir\" doesn't exist"
     [ ! -d "boards/$board" ] && error "$0: compile: error: board directory \"boards/$board\" doesn't exist"
 
     # compile
-    CCACHE_BASEDIR="$(pwd)" BOARD=$board RIOT_CI_BUILD=1 \
+    CCACHE_BASEDIR="$(pwd)" BOARD=$board TOOLCHAIN=$toolchain RIOT_CI_BUILD=1 \
```

Comment:
You'll need to add the TOOLCHAIN variable to the two make test calls below. Maybe just export TOOLCHAIN?

Reply:
Actually it should only be necessary for ```make test-murdock``` below.

Reply:
Why do test *runs* need a toolchain?

Reply:
I think because they are testing the executable produced by either gnu or llvm and only one could be failing.
Also they would be executed twice with the same name which can cause problems I imagine.

Reply:
Ok. That makes sense. I'll add it.